documentation/book/appendix_metrics.adoc

```diff
@@ -2,8 +2,8 @@
 [id='metrics-{context}']
 == Metrics
 
-This section describes how to monitor Strimzi Kafka and ZooKeeper clusters using Grafana dashboards.
-In order to run the example dashboards you must configure Prometheus server and add the appropriate https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter] rules to your Kafka cluster resource.
+This section describes how to monitor {ProductName} Kafka and ZooKeeper clusters using Grafana dashboards.
+To run the example dashboards you must configure a Prometheus server and add the appropriate xref:assembly-deployment-configuration-kafka-{context}[JMX exporter configuration] to your Kafka cluster resource.
```

Comment:
Does this link work properly when you render the docs (`make docu_html`)? I do not think it ill get you to the chapter about the `metrics` property.